atlas/templates/Makefile.common.gotmpl

```diff
@@ -60,30 +60,28 @@ fmt-atlas:
 test-atlas: fmt-atlas
 	@go test $(GO_TEST_FLAGS) $(GO_PACKAGES)
 
-docker-atlas:
-	@docker build -f $(SERVER_DOCKERFILE) -t $(SERVER_IMAGE):$(IMAGE_VERSION) .
+docker-atlas: .docker-$(IMAGE_NAME)-$(IMAGE_VERSION)
 	@docker image prune -f --filter label=stage=server-intermediate
 
 .docker-$(IMAGE_NAME)-$(IMAGE_VERSION):
-	$(MAKE) docker-atlas
+	@docker build -f $(SERVER_DOCKERFILE) -t $(SERVER_IMAGE):$(IMAGE_VERSION) .
```

Comment:
I found that usefull to build and push two images, image with strict tag and image with latest tag, what do you thing about adding this into atlas-cli ?